apps/app/src/app/pages/(components)/components/(breadcrumb)/breadcrumb--collapsed.example.ts

```diff
@@ -0,0 +1,123 @@
+import { Component } from '@angular/core';
+import {
+	HlmBreadcrumbComponent,
+	HlmBreadcrumbEllipsisComponent,
+	HlmBreadcrumbItemDirective,
+	HlmBreadcrumbLinkDirective,
+	HlmBreadcrumbListDirective,
+	HlmBreadcrumbPageDirective,
+	HlmBreadcrumbSeparatorComponent,
+} from '@spartan-ng/ui-breadcrumb-helm';
+
+@Component({
+	selector: 'spartan-breadcrumb-collapsed',
+	standalone: true,
+	imports: [
+		HlmBreadcrumbComponent,
+		HlmBreadcrumbSeparatorComponent,
+		HlmBreadcrumbEllipsisComponent,
+		HlmBreadcrumbListDirective,
+		HlmBreadcrumbItemDirective,
+		HlmBreadcrumbPageDirective,
+		HlmBreadcrumbLinkDirective,
+	],
+	template: `
+		<hlm-breadcrumb>
+			<ol hlmBreadcrumbList>
+				<li hlmBreadcrumbItem>
+					<a hlmBreadcrumbLink href="/home">Home</a>
+				</li>
+				<li hlmBreadcrumbItem>
+					<hlm-breadcrumb-separator />
+				</li>
+				<li hlmBreadcrumbItem>
+					<hlm-breadcrumb-ellipsis />
+				</li>
+				<li hlmBreadcrumbItem>
+					<hlm-breadcrumb-separator />
+				</li>
+				<li hlmBreadcrumbItem>
+					<a hlmBreadcrumbLink hlmL href="/components">Components</a>
```

Comment:
Is the directive `hlmL` used here or can it be removed?

Reply:
It is my mistake. I removed that wrong directive